react-basic-practice/src/components/Button/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { Button as ChakraButton, ButtonProps } from "@chakra-ui/react";
+import { borderRadius } from "../../themes/metrics";
+
+interface CustomButtonProps extends ButtonProps {
+  variant: "primary" | "secondary" | "default";
+  children: string | JSX.Element;
```

Comment:
pls change to label. Why we need jsx.element type here?

Reply:
i remove it